tools/Makefile.host

```diff
@@ -228,7 +228,7 @@ endif
 # jlink-nuttx - Create jlink GDBServer nuttx plugin
 
 jlink-nuttx$(HOSTDYNEXT): jlink-nuttx.c
-	$(Q) $(HOSTCC) $(HOSTCFLAGS) -shared jlink-nuttx.c -o jlink-nuttx$(HOSTDYNEXT)
+	$(Q) $(HOSTCC) $(HOSTCFLAGS) -shared -fPIC jlink-nuttx.c -o jlink-nuttx$(HOSTDYNEXT)
```

Comment:
maybe it should be added to `HOSTCFLAGS`?

Reply:
Are you sure about this? Everything else builds fine. Also `HOSTCFLAGS` is used for executables too, not only libraries.

Reply:
No. I'm not sure and just asking. So we are applying position independent code only for shared libraries? That makes sense.